discord-bot/src/register-commands.ts

```diff
@@ -0,0 +1,26 @@
+import { REST, Routes } from 'discord.js'
+import { config, commands } from './utils'
+
+//interact with API
+const rest = new REST({ version: '10' }).setToken(config.TOKEN)
+
+;(async () => {
```

Comment:
I learned something new 🏫 I saw this semi-colon at the start of the line and then noticed there are no semi-colons at end of lines, which then sent me to research this. I found some compelling arguments against end of line semi-colons in typescript such as [here](https://medium.com/@eugenkiss/dont-use-semicolons-in-typescript-474ccfe4bdb3).